pkg/eval/unix/umask.go

```diff
@@ -0,0 +1,94 @@
+// +build !windows,!plan9
+
+package unix
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"strconv"
+	"syscall"
```

Comment:
Do not use the `syscall` package; it's deprecated. Use golang.org/x/sys/unix instead.